README.md

```diff
@@ -1,23 +1,27 @@
-![](assets/ProgrammingBlockchain.png)
+# Programming The Blockchain in C# Community Edition
 
-[コードサンプルはこちら。](https://github.com/ProgrammingBlockchain/ProgrammingBlockchainCodeExamples/)
+![The logo of Programming The Blockchain in C# Community Edition](assets/ProgrammingBlockchain.png)
+
+[コードサンプルはこちら](https://github.com/ProgrammingBlockchain/ProgrammingBlockchainCodeExamples/)
 
 ## [この本を読むにはこちら](https://programmingblockchain.gitbooks.io/programmingblockchain-japanese/content/)
 
 ## コミュニティ
+
 [![Gitter](https://badges.gitter.im/MetacoSA/NBitcoin.svg)](https://gitter.im/MetacoSA/NBitcoin?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
 ## 他の言語版
+
 英語
 [Read](https://programmingblockchain.gitbooks.io/programmingblockchain/content/)- [GitHub](https://github.com/ProgrammingBlockchain/ProgrammingBlockchain) - [GitBook](https://www.gitbook.com/book/programmingblockchain/programmingblockchain)
 
 インドネシア語: [Read](https://nopara73.gitbooks.io/programmingblockchainindonesian/content/) - [GitHub](https://github.com/ProgrammingBlockchain/ProgrammingBlockchain-Indonesian) - [GitBook](https://www.gitbook.com/book/nopara73/programmingblockchainindonesian)
 
-## 筆者にフィードバックしたいときは
+## フィードバックするためには
 
-もし読んでいて間違いを発見してご自分で修正しない場合は[Github](https://github.com/ProgrammingBlockchain/ProgrammingBlockchain-Japanese)で課題を提起してください。もしGitBookで読んでいる場合は、インラインでコメントすることも可能です。
+[Github](https://github.com/ProgrammingBlockchain/ProgrammingBlockchain-Japanese)で Issue を作成し、課題を提起してください。もちろんオープンソースですので、自ら修正していただくことは大歓迎です！もしGitBookで読んでいる場合は、インラインでコメントすることも可能です。
```

Comment:
@hypergori Yes I agree. I wanted many people to improve this repository so that this translation version is not going to be outdated. But this is my opinion. Will remove this.